src/test/java/org/stellar/sdk/Sep10ChallengeTest.java

```diff
@@ -1608,6 +1608,43 @@ public void testVerifyChallengeTransactionThresholdValidServerAndMultipleClientK
     assertEquals(new HashSet<String>(Arrays.asList(masterClient.getAccountId(), signerClient1.getAccountId())), signersFound);
   }
 
+  @Test
+  public void testVerifyChallengeTransactionThresholdValidServerAndMultipleClientKeyMeetingThresholdSomeUnusedAndOneNotEd25519Compliant() throws IOException, InvalidSep10ChallengeException {
```

Comment:
now that's a test case name, good coverage to assert the change! maybe condense the name down to just the non-compliant aspect.